fitbenchmarking/fitting/scipy/func_def.py

```diff
@@ -27,24 +27,25 @@
 import numpy as np
 import re
 
-from fitting.scipy.nist_def import nist_func_definitions
-from fitting.scipy.neutron_def import neutron_func_definitions
+from fitting.scipy.dat_functions import dat_func_definitions
```

Comment:
@wathen please remove all references of dat and txt and replace with NIST (or nist) and FitBenchmark (or fitbenchmark or native) since dat and and txt do not contain information about content but nist and fitbenchmark/native, i.e. the former follow the nist format and the latter the format native to this application